src/main/java/com/hillel/items_exchange/controller/CategoryController.java

```diff
@@ -93,12 +88,12 @@ public CategoryDto createCategory(@Validated(New.class) @RequestBody CategoryDto
 
     @PreAuthorize(HAS_ROLE_ADMIN)
     @PutMapping
-    @ResponseStatus(HttpStatus.ACCEPTED)
-    @ApiOperation(value = "Update an existent category. Only for users authorized as admin.")
+    @ApiOperation(value = "Update an existed category", notes = "ADMIN ONLY")
     @ApiResponses(value = {
-            @ApiResponse(code = 201, message = "CREATED"),
+            @ApiResponse(code = 202, message = "ACCEPTED"),
```

Comment:
Maybe it's better to return 204 here?

Reply:
It's not the right choice for Update response code. 202 is that what we really wanted here.